test/test_helpers/indexer.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literals: true
+
+module Indexer
+  DEFAULT_OPTIONS = { verbose: false, wait_until_finished: true }.freeze
+  MAX_WAITING_CYCLES = 100.freeze
```

Comment:
You can't freez numbers they are already :)

Reply:
LOL. Sorry, good bad habits.

Reply:
Fixed

Reply:
Actually you can! 😮 It just noop!